### PR TITLE
Update sharepoint-site-usage.md

### PR DESCRIPTION
**Pull request by amol-karkhanis**
Fixed two "note" fields and spelling

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/activity-reports/sharepoint-site-usage.md [hunk 1]
```diff
@@ -52,9 +52,9 @@ As an Microsoft 365 admin, the **Reports** dashboard shows you the activity over
...
+|10.  <br/> |You can also export the report data into an Excel .csv file, by selecting the **Export** ![Export](../media/4dc548cc-8061-48d5-9240-6793affca43a.png) link. This exports data for all sites and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 sites, you can sort and filter within the table in the report itself. If you have more than 2000 sites, in order to filter and sort, you will need to export the data.  <br/> NOTE: When the data is exported to an Excel file, note that the date the content report was generated in reflected in the file in the **Data as of** column.      <br/>   |
```
Comment (JohanFreelancer9) [inline Office365-Admin/activity-reports/sharepoint-site-usage.md]:
```suggestion
|10.  <br/> |You can also export the report data into an Excel .csv file, by selecting the **Export** ![Export](../media/4dc548cc-8061-48d5-9240-6793affca43a.png) link. This exports data for all sites and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 sites, you can sort and filter within the table in the report itself. If you have more than 2000 sites, in order to filter and sort, you will need to export the data.  <br/> NOTE: When the data is exported to an Excel file, note that the date the content report was generated is reflected in the file in the **Data as of** column.      <br/>   |
```

**Comment by mypil** [comment]
@amol-karkhanis - please merge the commit on the copy/edit review made by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@kaarins - PR has been copy edited and is ready for final review.

Thank you

cc: @kenwith

**Pull request merged**